posts/navarroaxel/19.git-rebase-exec.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Git rebase exec to validate every commit
+published: true
+description: How to run unit tests and build tools for every commit when you rebase
+tags: 'git, bash, productivity, tooling'
+cover_image: ./assets/git-rebase-exec.png
+id: 1949938
+---
+
+In the previous post we checked out how the `git rebase` works and how to use the `--interactive` mode to keep your topic branch up-to-date with the target (e.g. `main`). Adding the `exec` command to run the tests and lint tools to check every rebased commit.
+
+Now, let's check how to use`exec` in a short way.
+
+## Exec, exec, exec
+
+If you need to run the build, tests, or a format tool when you are rebasing because you must guarantee that every commit is a valid commit, you can speed it up using the `git rebase --exec` command.
```

Comment:
```suggestion
If you need to run the build, tests, or any format tool while rebasing because you must guarantee that every commit is valid, you can speed things up using the `git rebase --exec` command.
```